src/core/init.spec.ts

```diff
@@ -0,0 +1,27 @@
+import {assert} from 'chai';
+import * as Relution from './init';
+
+describe(module.filename || __filename, function() {
+  it ('if serverUrl fails', (done) => {
+    const options = {
+      serverUrl: 'ashdhasdasd/gsdvhasd'
+    };
+    return Relution.init(options).catch((e) => {
+      assert(e.message.indexOf(options.serverUrl) !== -1, `Url fails ${e.message}`);
+      done();
```

Comment:
1. I found that mocha supports promises directly! The tests become easier when instead of (done) => ... you just do a () => ... and make that return a promise as you do already. Mocha detects the case when the function does not take a done callback but returns a promise and it waits automatically for the promise resolving/rejecting. That's pretty need, or? Notice, I changed all our other tests to do it like that. Please adjust your code accordingly.
2. I think this test must ensure the test fails when the init() call resolves! I mean, it should pass only if the catch code path is taken. Please add a then() executing assert.fail().

Here's what I mean:
```

   it ('if serverUrl fails', () => { // NO done HERE!
     const options = {
       serverUrl: 'ashdhasdasd/gsdvhasd'
     };
     return Relution.init(options).then(() => {
             throw new Error('expected to fail!');
       }, (e) => {
       assert(e.message.indexOf(options.serverUrl) !== -1, `Url fails ${e.message}`);
       return true; // just signals everything is fine
      });
```